src/core/external/collect/collect-container.ts

```diff
@@ -29,26 +33,46 @@ import {
 import Container from '../common/container';
 import CollectElement from './collect-element';
 
+/** Type of Skyflow Element. */
 export interface CollectElementInput {
+  /** Table this data belongs to */
   table?: string;
+  /** Data should be inserted into the column. */
   column?: string;
+  /** Styles should be applied to the form element. */
   inputStyles?: object;
+  /** Label for the form element. */
   label?: string;
+  /** Styles applied to the label of the collect element. */
   labelStyles?: object;
+  /** Styles that apply to the error text of the collect element will be added. */
   errorTextStyles?: object;
+  /** A placeholder for the form element. */
   placeholder?: string;
+  /** Skyflow.ElementType enum */
   type: ElementType;
+  /** It string acts as an initial value for the Collect Element. */
   altText?: string;
+  /** A set of validations for Collect Element. */
   validations?: IValidationRule[]
+  /** SkyflowID fo the element. */
   skyflowID?: string;
 }
 
-interface ICollectOptions {
+/** Wraps the options for the Collect method. */
+export interface ICollectOptions {
+  /** Indicates whether to return tokens for the collected data. It defaults to 'true'. */
   tokens?: boolean;
+  /** Insert the non-PCI elements data into the vault in the format of the records object. */
   additionalFields?: IInsertRecordInput;
+  /** To support upsert operations while collecting data from Skyflow elements, you should pass the table and column marked as unique in the table. */
```

Comment:
"[Upsert](https://docs.skyflow.com/unique-columns-upsert) configuration for the element."